frontend/gateway/pb/gateway.proto

```diff
@@ -92,6 +95,29 @@ message ReadFileResponse {
 	bytes Data = 1;
 }
 
+message ReadDirRequest {
+	string Ref = 1;
+	string DirPath = 2;
+	string IncludePattern = 3;
+}
+
+enum StatResultKind {
```

Comment:
These seem to be unused now

Reply:
ack, I only checked the go files. Nuking now.